go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/tektoncd/pipeline
 go 1.16
 
 require (
+	github.com/abayer/gen-crd-api-reference-docs v0.999.0
```

Comment:
Could you comment on why we need your fork here? 
Is there a chance we can eventually switch back to the original package?

Reply:
Good point! On it.